bokeh/models/ui/__init__.py

```diff
@@ -0,0 +1,46 @@
+#-----------------------------------------------------------------------------
+# Copyright (c) 2012 - 2022, Anaconda, Inc., and Bokeh Contributors.
+# All rights reserved.
+#
+# The full license is in the file LICENSE.txt, distributed with this software.
+#-----------------------------------------------------------------------------
+"""
+
+"""
+#-----------------------------------------------------------------------------
+# Boilerplate
+#-----------------------------------------------------------------------------
+from __future__ import annotations
+
+import logging # isort:skip
+log = logging.getLogger(__name__)
+
+#-----------------------------------------------------------------------------
+# Imports
+#-----------------------------------------------------------------------------
+
+# Bokeh imports
+from .inspector import *
+from .tooltips import *
+
+#-----------------------------------------------------------------------------
+# Globals and constants
+#-----------------------------------------------------------------------------
+
+# __all__ = include all explicit transitive imports above
```

Comment:
Why not just concatenate `inspector.__all__ + tooltips.__all__` ? Then I don't think you will need to silence LGTM

Reply:
I will try that. So far I've been copying the code for the existing solution.

Reply:
This actually works.